Change version id to 0.1-ubuntu1
Index: purple-line/libpurple/pluginmain.cpp
===================================================================
--- purple-line.orig/libpurple/pluginmain.cpp
+++ purple-line/libpurple/pluginmain.cpp
@@ -34,7 +34,7 @@ static void init_info(PurplePluginInfo &
 
     i.id = (char *)LINE_PRPL_ID;
     i.name = (char *)"LINE";
-    i.version = (char *)"0.1";
+    i.version = (char *)"0.1-ubuntu1";
     i.summary = (char *)"Plugin for Naver LINE";
     i.description = (char *)"Plugin for Naver LINE";
     i.author = (char *)"Matti Virkkunen <mvirkkunen@gmail.com>";
